numpy/lib/tests/test_function_base.py

```diff
@@ -1089,6 +1089,29 @@ def test_array_like(self):
         assert_array_equal(y1, y2)
         assert_array_equal(y1, y3)
 
+    def test_scale(self):
+        x = np.linspace(-1, 1, 100)
+
+        # Check backwards-compatibility of default
+        assert_array_equal(sinc(x, 'normalized'), sinc(x))
+
+        # Check unnormalized
+        assert_equal(sinc(0, 'unnormalized'), 1)
+        assert_almost_equal(sinc(np.pi, scale='unnormalized'), 0)
+        assert_almost_equal(sinc(np.pi / 2.0, 'unnormalized'), 2.0 / np.pi)
+
+        # Check scalar
+        assert_equal(sinc(0, 2), 1)
+        assert_almost_equal(sinc(np.pi / 2.0, 2.0), 0)
+        assert_almost_equal(sinc(np.pi / 4.0, scale=2.0), 2.0 / np.pi)
+
+        # Check negative
+        assert_equal(sinc(0, -5), 1)
+        assert_equal(sinc(x, -5.0), sinc(x, scale=5.0))
+
+        # Check bad names
+        assert_raises(ValueError, sinc, x, 'foo')
+        assert_raises(ValueError, sinc, x, scale = 'raw')
```

Comment:
Can `scale` broadcast against `x`? We should probably decide one way or the other and add a test for whatever we decide.